ontology/oba.md

```diff
@@ -28,12 +28,50 @@ products:
 - id: oba.obo
 - id: oba/subsets/oba-basic.obo
 publications:
-- id: https://doi.org/10.1101/2023.01.26.525742
-  title: The Ontology of Biological Attributes (OBA) - Computational Traits for the Life Sciences
+- id: https://doi.org/10.1007/s00335-023-09992-1
+  title: The Ontology of Biological Attributes (OBA) - computational traits for the life sciences
 repository: https://github.com/obophenotype/bio-attribute-ontology
 tracker: https://github.com/obophenotype/bio-attribute-ontology/issues
+usages:
+- description: OBA terms are used by the NHGRI-EBI GWAS Catalog for phenotypic trait annotation.
+  examples:
+  - description: age of onset of depressive disorder
+    url: https://www.ebi.ac.uk/gwas/efotraits/OBA_2040166
+  publications:
+  - id: https://doi.org/10.1093/nar/gkac1010
+    title: 'The NHGRI-EBI GWAS Catalog: knowledgebase and deposition resource'
+  type: annotation
+  user: https://www.ebi.ac.uk/gwas/
+- description: OBA trait terms facilitate computational drug target identification via the Open Targets Platform.
+  examples:
+  - description: age of onset of Alzheimer disease
+    url: https://platform.opentargets.org/disease/OBA_2001000/associations
+  publications:
+  - id: https://doi.org/10.1093/nar/gkac1046
+    title: 'The next-generation Open Targets Platform: reimagined, redesigned, rebuilt'
+  type: Database
+  user: https://platform.opentargets.org
+- description: The Encyclopedia of Life (EOL) TraitBank takes advantage of the well-axiomatised OBA terms to infer traits in biodiversity data and to improve their search functionality.
+  examples:
+  - description: body mass http://purl.obolibrary.org/obo/OBA_VT0001259 The amount of matter in the body of an organism.
+    url: https://eol.org/terms/search_results?tq%5Bf%5D%5B0%5D%5Bp%5D=260&tq%5Br%5D=record
```

Comment:
For the record, this example is not too good, this one is better: 
https://eol.org/terms/search_results?tq%5Bf%5D%5B0%5D%5Bp%5D=380&tq%5Br%5D=record

because it clearly shows an OBA use. But this is proof enough for me.

Reply:
Yes, 'cell size' is a better example. I will use that.